feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -138,10 +140,50 @@ fun ElapsedTimeText(
     }
 }
 
+@Composable
+fun PauseResumeToggleButton(
+    modifier: Modifier = Modifier,
+    onSetPause: (Boolean) -> Unit,
+    size: Int = 75,
```

Comment:
I think size is only used as Float type, what about change type of size to Float?